support/GhidraVersion.cpp.in

```diff
@@ -0,0 +1,17 @@
+/*
+  Copyright (c) 2022-present, Trail of Bits, Inc.
+  All rights reserved.
+
+  This source code is licensed in accordance with the terms specified in
+  the LICENSE file found in the root directory of this source tree.
+*/
+
+#include "Version.h"
+
+namespace sleigh {
+
+std::string_view GetGhidraVersion(void) { return "@ghidra_version@"; }
+
+std::string_view GetGhidraCommitHash(void) { return "@ghidra_git_tag@"; }
+
+} // namespace sleigh
```

Comment:
I think we should also let the user know whether the Ghidra commit they're using is `stable` or `HEAD`, which should be obtained through `SLEIGH_GHIDRA_RELEASE_TYPE`.

Reply:
e978408